AjobK/Athena
# 350 add a tooltip

closes #350

<br />

 **What is it supposed to do**  
Whenever you are creating a post, or editing a post a tooltip needs to appear after selecting some text. With this tooltip you can apply some markdown to your written text. For mobile and tablet devices a mobile bar should appear which needs to be scrollable if there are a lot of buttons. It has the same behaviour as the tooltip, which is applying markdown to text.

The icon component got refactored. If you use a prefix, and an iconname which is found on: https://mui.com/components/material-icons/. If you have a custom svg you can use that too by using the prefix 'custom'. You then give the iconName the image file, which will eventually render it on your screen.

<br />

 **How can we test your branch** 
1. Create a post
2. Write some text
3. Select text

It should look like this: 
![image](https://user-images.githubusercontent.com/55099336/147552301-e9481e12-fcb4-4618-89db-6cd04e02b8bf.png)

And for mobile it should look like this:
![image](https://user-images.githubusercontent.com/55099336/147552495-e98905ac-cf9d-420d-8911-bf08b5cc3b59.png)

If the icons are appearing the icon component using MUI works. Inspect code to further examine the component.

<br />

 **Additional notes**  
- Adding active to the mobile bar was to broad for this feature, because the mobile bar component is a generic component which will be used for mobile/tablet devices.

<br />

**1. General checklist**

<br />

**2. If relevant, front-end checklist**

<br />

**3. If relevant, back-end checklist**

<br />

**4. If relevant, test these browsers**

---

**PR Rules**  
*1. Add relevant labels.*  
*2. Select at least two reviewers.*  
*3. Assign all reviewers allowed to merge the branch, including yourself.*  
*4. If relevant to the sprint, put related issue in 'Awaiting Approval' column on project board.*  
*5. Add to the corresponding [milestone](https://github.com/AjobK/Seaqull/milestones).*

<br />

**Reviewer info**  
*1. You can use aliases to refer to specific PR checklist items. For example 'Hey @JohnDoe, for this line please check 3.a and 1.d.'*  
*2. Where possible, please use the suggestions feature in github so the author has a better idea of the solution you had in mind.*   

<br />

> This form should be filled in by the author of this PR.
